lib/msal-browser/src/app/Configuration.ts

```diff
@@ -77,12 +81,12 @@ export type BrowserLoggerOptions = {
  * - framework: this is where you can configure the running mode of angular. More to come here soon.
  */
 export type Configuration = {
-    auth?: AuthOptions,
+    auth?: BrowserAuthOptions,
```

Comment:
Something like navigateToLoginRequestUrl is something that wouldn't be present on a node app, so i thought it would make sense to do this. Open to suggestions if there is feedback on this.